src/Paket.Core/PackageResolver.fs

```diff
@@ -189,6 +189,7 @@ type Resolved = {
 type UpdateMode =
     | UpdatePackage of GroupName * PackageName
```

Comment:
can we remove that case and express it as `UpdateFiltered of GroupName * PackageFilter`?